diff --git a/build/src/start/platforms/android/adb.js b/build/src/start/platforms/android/adb.js
index 608e5b02733951e04f336efda798e263d73cc9ba..d8470e7918302492c00f3f56db5431fa978a1809 100644
--- a/build/src/start/platforms/android/adb.js
+++ b/build/src/start/platforms/android/adb.js
@@ -190,7 +190,7 @@ async function launchActivityAsync(device, { launchActivity, url }) {
     return openAsync(adbArgs(device.pid, ...args));
 }
 async function openAppIdAsync(device, { applicationId }) {
-    return openAsync(adbArgs(device.pid, 'shell', 'monkey', '-p', applicationId, '-c', 'android.intent.category.LAUNCHER', '1'));
+    return openAsync(adbArgs(device.pid, 'shell', 'monkey', '--pct-syskeys', '0', '-p', applicationId, '-c', 'android.intent.category.LAUNCHER', '1'));
 }
 async function openUrlAsync(device, { url }) {
     return openAsync(adbArgs(device.pid, 'shell', 'am', 'start', '-a', 'android.intent.action.VIEW', '-d', // ADB requires ampersands to be escaped.
